docs/source/fundamentals/tips.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Tips for success
+description: Your starting point for learning GraphQL and Apollo
+---
+
+TK WIP TODO finish this intro.  Depending on the size of an organization, there are usually existing practices surrounding the management of the data 
+
+<h2 id="incremental-adoption">Incremental adoption</h2>
+
+While it makes an excellent longer-term goal, GraphQL doesn’t need to stand above the entire data tree in an organization in order to provide value and expending large amounts of engineering effort to switch an entire stack should generally be avoided.
+
+Unlike some technologies, which are difficult to embrace without getting buy-in from all the teams in an organization, GraphQL can be utilized by smaller units (e.g. component by component, team by team) and later integrated into the bigger picture.
+
+Thanks to GraphQL’s ability to fetch data from any data source, it makes sense to keep existing APIs as they are and use GraphQL to “wrap” the data.  This can be appreciated by organizations with existing collections of REST (or similar) APIs which power their user experiences since they can lean heavily on their existing investments.  This allows small teams to immediately realize many benefits of GraphQL without waiting for deeper adoption and allows existing versions of the application, which have already been deployed and rely on the existing API, to keep functioning uninterrupted.
+
+<h2 id="schema">Schema scope and ownership</h2>
+
+APIs change as the products which use them evolve and their ideal structure is usually defined by the front-end applications which consume them.  While it’s certainly possible, and sometimes desirable, to have a single GraphQL schema which blankets an entire organization’s operational concerns, trying to manage a large schema presents challenges which are avoided by separating the schema into more manageable pieces that align with individual products.
+
+Luckily, the GraphQL Schema Definition Language (SDL) focuses on the shape of the data and not the actual implementation, allowing product teams to be the ideal custodians of their own GraphQL schemas.
+
+Having product teams own the GraphQL schema for their products allows the schema to act as an API contract within the team and enhances intra-team communication by providing clearly defined expectations.  These teams can utilize schema-driven development which lets them divide back-end and front-end work, mocking various parts of the API until full functionality is achieved, while remaining confident that they will arrive back at a unified solution.
```

Comment:
It would be great if we could link to that article from the NYT CTO here where he admits that they should have started with the product when they first started transitioning to GraphQL. I think having a qualified voice to back up our claims would make this section even stronger.

Reply:
Does anyone have a link handy for that article? I can't seem to find it.